core/src/test/java/io/snyk/plugins/artifactory/scanner/ScannerModuleTest.java

```diff
@@ -74,6 +77,10 @@ ScanTestSetup createScannerSpyModuleForTest(
   ) throws Exception {
     SnykConfig config = customiseBuilder.apply(SnykConfigForTests.newBuilder()).build();
     Properties properties = new Properties();
+    properties.put(SCANNER_PACKAGE_TYPE_MAVEN.propertyKey(), "true");
+    properties.put(SCANNER_PACKAGE_TYPE_NPM.propertyKey(), "true");
+    properties.put(SCANNER_PACKAGE_TYPE_PYPI.propertyKey(), "true");
```

Comment:
Making sure all scanners are enabled for ScannerModule tests, regardless of defaults.